### PR TITLE
id fixes

### PR DESCRIPTION
**Pull request by riesters**
为我的失败祈祷             (✿◡‿◡)

### CODE REVIEW

File: Horion/Module/Modules/AutoClicker.cpp [hunk 1]
```diff
@@ -22,8 +22,9 @@ void AutoClicker::onTick(C_GameMode* gm) {
 		Odelay++;
 
 		if (Odelay >= delay) {
-			auto selectedItem = localPlayer->getSelectedItem();
-			if (weapons && selectedItem->getAttackingDamageWithEnchants() < 1)
```
Comment (N0tExisting) [inline Horion/Module/Modules/AutoClicker.cpp:26]: this should still be checked

File: Horion/Module/Modules/BowAimbot.cpp [hunk 2]
```diff
@@ -42,8 +42,8 @@ void BowAimbot::onPostRender(C_MinecraftUIRenderContext* renderCtx) {
 	C_LocalPlayer* localPlayer = g_Data.getLocalPlayer();
 	if (localPlayer == nullptr)
 		return;
-
-	if (localPlayer->getSelectedItemId() != 261)  // Bow in hand?
+	auto selectedItemId = localPlayer->getSelectedItemId();
+	if (!(selectedItemId == 300 || selectedItemId == 565 || selectedItemId == 536))  // Item in hand?
 		return;
 
 	if (!(GameData::isRightClickDown() && GameData::canUseMoveKeys())) // is aiming?
```
Comment (N0tExisting) [inline Horion/Module/Modules/BowAimbot.cpp:49]: If there is a crossbow in hand only return when the user is holding down rightclick, but not when he isn't.

**Review by N0tExisting — COMMENTED** [review]
Nice, could use some improvement

**Comment by MrDe0** [comment]
where should i write an issue ?

**Comment by grajkogj** [comment]
> where should i write an issue ?

the issues tab was taken down a while ago due to spamming

**Comment by MrDe0** [comment]
all right then is asking for a StackableItem fix Considered spamming?
I need at least some advice on how to fix it.
i am not that good at c++ and injection stuff, i just have a general idea on how it works.

**Comment by N0tExisting** [comment]
> all right then is asking for a StackableItem fix Considered spamming?
> I need at least some advice on how to fix it.
> i am not that good at c++ and injection stuff, i just have a general idea on how it works.

Just try out some simpler stuff first if u aren't confident

**Comment by MrDe0** [comment]
all right will do👍
 but i found the new version is much harder at getting pointers isn't that right?

**Comment by MrDe0** [comment]
Also do you advise me to suffice with the horizon src or use cheat engine to analyze the game?

**Pull request closed without merging**